spec/zoneddatetime.html

```diff
@@ -1187,13 +1222,26 @@ <h1>CreateTemporalZonedDateTime ( _epochNanoseconds_, _timeZone_, _calendar_ [ ,
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-temporalzoneddatetimetostring" aoid="TemporalZonedDateTimeToString">
-      <h1>TemporalZonedDateTimeToString ( _zonedDateTime_, _precision_, _showCalendar_, _showTimeZone_, _showOffset_ [ , _increment_, _unit_, _roundingMode_ ] )</h1>
-      <p>
-        The abstract operation TemporalZonedDateTimeToString returns an ISO 8601 string representation of its argument, including a time zone name annotation and calendar annotation, which are extensions to the ISO 8601 format.
-      </p>
+    <emu-clause id="sec-temporal-temporalzoneddatetimetostring" type="abstract operation">
+      <h1>
+        TemporalZonedDateTimeToString (
+          _zonedDateTime_: a Temporal.ZonedDateTime,
+          _precision_: a String or integer,
```

Comment:
This is probably OK, but we could also make it even more specific, e.g. "one of **"auto"**, **"minute"**, or an integer between 0 and 9 inclusive".
Ditto for the other options below, e.g. `showCalendar` could be "one of **"auto"**, **"always"**, or **"never"**".

Reply:
Done :+1: